starfish/intensity_table/intensity_table.py

```diff
@@ -389,10 +394,29 @@ def from_image_stack(
         return IntensityTable.from_spot_data(intensity_data, pixel_coordinates)
 
     @staticmethod
-    def concatanate_intensity_tables(intensity_tables: List["IntensityTable"]):
-        # TODO VARY CONCAT LOGIC IF TILES OVERLAP
-        # This method is a starting point for handling tile overlap, right now
-        # it does a simple concat but people want other overlap logic implmented
+    def process_overlaps(its: List["IntensityTable"],
+                         overlap_strategy: OverlapStrategy
+                         ) -> List["IntensityTable"]:
+        """Find the overlapping sections between IntensityTables and process them according
+        to the given overlap strategy
+        """
+        overlap_pairs = find_overlaps_of_xarrays(its)
+        for indices in overlap_pairs:
+            overlap_method = OVERLAP_STRATEGY_MAP[overlap_strategy]
+            idx1, idx2 = indices
+            # modify IntensityTables based on overlap strategy
+            it1, it2 = overlap_method(its[idx1], its[idx2])
```

Comment:
I think this strategy might break down where you have three intensity tables that overlap in one area.  It might be easier to illustrate this if I'm not making sense.